data_prep/england/get_input.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+cd tmp
+
+function od {
+  # From https://www.nomisweb.co.uk/sources/census_2021_od
+  wget https://www.nomisweb.co.uk/output/census/2021/odwp01ew.zip
+  unzip odwp01ew.zip
+
+  # Filter for OD pairs starting and ending in England
+  xan filter 'startswith(col("Output Areas code"), "E") && startswith(col("OA of workplace code"), "E")' ODWP01EW_OA.csv | \
```

Comment:
I've been using xsv before, but apparently it's deprecated, and https://github.com/medialab/xan is one replacement. It does a few useful things and is quick to learn.

Reply:
Looks nice!